README.md

```diff
@@ -68,8 +74,8 @@ module.exports = {
 };
 ```
 
-NOTE: If your primary TSConfig file is at the project root and named `tsconfig.json` *you need to overwrite the parser
-options to point to your config file*.
+NOTE: If your primary TSConfig file is *NOT* at the project root and named `tsconfig.json` *you need to overwrite the
```

Comment:
Totally different meaning now lol

Reply:
Yeah lol that was a straight up error on my part 🤦‍♂